tests/e2e/ansible/install_test_deps.yml

```diff
@@ -37,7 +37,7 @@
       ignore_errors: yes
     - name: Install kustomize
       shell: |
-        curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+        curl -s --retry 3 --retry-delay 10 "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
Luckily the curl version installed on CI supports `--retry`?  I think it was a patch to kata that it didn't work because it has used an old curl, right?

Reply:
The `--retry` option is fairly old, the problem in kata was the `--retry-on-all-errors`